packages/google_identity_services_web/example/integration_test/js_interop_id_test.dart

```diff
@@ -37,45 +40,46 @@ void main() async {
       final IdConfiguration config = IdConfiguration(
         client_id: 'testing_1-2-3',
         auto_select: false,
-        callback: allowInterop((_) {}),
+        callback: (_) {},
         login_uri: Uri.parse('https://www.example.com/login'),
-        native_callback: allowInterop((_) {}),
+        native_callback: (_) {},
         cancel_on_tap_outside: false,
         prompt_parent_id: 'some_dom_id',
         nonce: 's0m3_r4ndOM_vALu3',
         context: OneTapContext.signin,
         state_cookie_domain: 'subdomain.example.com',
         ux_mode: UxMode.popup,
         allowed_parent_origin: <String>['allowed', 'another'],
-        intermediate_iframe_close_callback: allowInterop((_) {}),
+        intermediate_iframe_close_callback: () {},
         itp_support: true,
         login_hint: 'login-hint@example.com',
         hd: 'hd_value',
         use_fedcm_for_prompt: true,
       );
 
-      // Save some keystrokes below by partially applying to the 'config' above.
-      void expectConfigValue(String name, Object? matcher) {
-        expect(js_util.getProperty(config, name), matcher, reason: name);
-      }
+      final utils.ExpectConfigValueFn expectConfigValue =
+          utils.createExpectConfigValue(config as JSObject);
 
-      expectConfigValue('allowed_parent_origin', hasLength(2));
+      expectConfigValue('client_id', 'testing_1-2-3');
       expectConfigValue('auto_select', isFalse);
-      expectConfigValue('callback', isA<Function>());
+      expectConfigValue('callback', isA<JSFunction>());
```

Comment:
Side note: `is` is dependent on the compiler since it's a runtime check. This is fine because you only use JS types for the `is` check, and dart2js/ddc are more restrictive than dart2wasm there. So if it passes on dart2js/ddc, this is fine, but something to be aware.

Reply:
Would this pass in dart2wasm? I eventually want to check there too. Is there any way of checking `typeof obj === "function"` that works across all compilers?

Reply:
Yes, but I think the the right way is to use `typeofEquals` e.g. `Expect.isTrue(callback.typeofEquals('function'), true);` since you want expectations on the JS value itself and not our representation of it (which may differ).  I have a proposal to add a `isA` member that makes this less annoying to write: https://github.com/dart-lang/sdk/issues/54138.